src/pages/login.js

```diff
@@ -50,7 +50,7 @@ function Login() {
             id="email"
             name="email"
             type="text"
-            className="bg-white rounded-md w-[422px] h-[38px] hover:border-4 hover:border-blue-login active:border-4 active:border-blue-login placeholder:text-gray p-2"
+            className="bg-white text-sm rounded-md w-[422px] h-[38px] hover:border-4 hover:border-blue-login active:border-4 active:border-blue-login placeholder:text-gray p-2"
```

Comment:
Don't know if this was request by Design, but note than this is 14px and in mobile, forms will auto zoom to an input if the text size is below 16px, so it should't really go below 16px for an input, especially.

Reply:
Yes it was, I'll ask Natalia about it